src/providers/sambanova/api.ts

```diff
@@ -0,0 +1,19 @@
+import { ProviderAPIConfig } from '../types';
+
+const SambaNovaAPIConfig: ProviderAPIConfig = {
+  getBaseURL: ({ providerOptions }) =>
+    providerOptions.urlToFetch || 'https://fast-api.snova.ai/v1',
```

Comment:
Hi. If the idea here is to allow dedicated deployments, then `customHost` would be the correct parameter to use instead of `urlToFetch`